packages/jest-config/src/normalize.js

```diff
@@ -148,6 +152,17 @@ const setupBabelJest = (options: InitialOptions) => {
         babelJest = customJSTransformer;
       }
     }
+
+    if (customTSPattern) {
```

Comment:
not sure about this one. Is this correct?

Reply:
couldn't we just extend the `customJSPattern` check?
```js
     const customPattern = Object.keys(transform).find(pattern => {
       const regex = new RegExp(pattern);
       return regex.test('a.js') || regex.test('a.jsx') || regex.test('a.ts') || regex.test('a.tsx');
     });
```

Reply:
Ok, that doesn't make sense

Reply:
what if there are different patterns for `js` and `ts`, such as `ts-jest`?